gtfs/spec/en/reference.md

```diff
@@ -132,8 +133,8 @@ File: **Required**
 |  ------ | ------ | ------ |
 |  route_id | **Required** | The **route_id** field contains an ID that uniquely identifies a route. The route_id is dataset unique. |
 |  agency_id | Optional | The **agency_id** field defines an agency for the specified route. This value is referenced from the [agency.txt](#agencytxt) file. Use this field when you are providing data for routes from more than one agency. |
-|  route_short_name | **Required** | The **route_short_name** contains the short name of a route. This will often be a short, abstract identifier like "32", "100X", or "Green" that riders use to identify a route, but which doesn't give any indication of what places the route serves. At least one of *route_short_name* or *route_long_name* must be specified, or potentially both if appropriate. If the route does not have a short name, please specify a *route_long_name* and use an empty string as the value for this field. |
-|  route_long_name | **Required** | The **route_long_name** contains the full name of a route. This name is generally more descriptive than the *route_short_name* and will often include the route's destination or stop. At least one of *route_short_name* or *route_long_name* must be specified, or potentially both if appropriate. If the route does not have a long name, please specify a *route_short_nam*e and use an empty string as the value for this field. |
+|  route_short_name | **Required** | The **route_short_name** contains the short name of a route. This will often be a short (6 characters or less), abstract identifier like "32", "100X", or "Green" that riders use to identify a route, but which doesn't give any indication of what places the route serves. At least one of *route_short_name* or *route_long_name* must be specified. If the route does not have a short name, please specify a *route_long_name* and use an empty string as the value for this field. |
```

Comment:
"TGV" is fair use of this field if local riders identify the same route (without 
confusing another) as "TGV". 

Identification of route by number: "32" is certainly common practice in UK, Singapore, etc.
"Green" and "Red" are also often to identify tube lines in a city.

As such, "TGV" is acceptable use given context but not sure if ideal sample use case.